cli/prisma2/src/bin.ts

```diff
@@ -91,7 +99,20 @@ async function main(): Promise<number> {
     console.error(result)
     return 1
   }
-  console.log(result)
+
+  const checkResult = await checkpoint.check({
+    product: 'prisma',
+    version: packageJson.version,
+    cache_only: true,
+    disable: isCI,
+  })
+  if (checkResult && checkResult.outdated) {
+    console.error(
+      `\n${chalk.blue('Update available')} ${packageJson.version} -> ${checkResult.current_version}\nRun ${chalk.bold(
+        'npm install -g prisma2',
```

Comment:
This command does not work for `alpha`.

Reply:
good call, it makes the most sense for this to come from the API, so i added it to https://checkpoint.prisma.io.